src/patterns/dosdp-patterns/entity_attribute_partOf.yaml

```diff
@@ -0,0 +1,68 @@
+pattern_name: entity_attribute
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/entity_attribute.yaml
+
+description: '
+
+  Basic pattern for traits that captures a quality, such as size or rate which is a
```

Comment:
Is this intended as a basic pattern for traits? I thought this template was intended to help create a carefully selected set of grouping terms.
The description hear should provide some guidance on when to use this pattern instead of the basic `entity_attribute` pattern.